packages/trader/src/AppV2/Components/Guide/decription-modal.tsx

```diff
@@ -0,0 +1,116 @@
+import React from 'react';
+import { ActionSheet, Heading, Chip, Text } from '@deriv-com/quill-ui';
+import { Localize } from '@deriv/translations';
+import { observer, useStore } from '@deriv/stores';
+import { clickAndKeyEventHandler } from '@deriv/shared';
+import { AVAILABLE_CONTRACTS, CONTRACT_LIST } from 'AppV2/Utils/trade-types-utils';
+import { getDescriptionVideoIds } from 'AppV2/Utils/contract-description-utils';
+import VideoPlayer from 'App/Components/Elements/VideoPlayer';
+import TradeDescription from './Description/trade-description';
+import VideoPreview from './Description/video-preview';
```

Comment:
description-modal.tsx

the filename is missing an s

Reply:
Would be good to have more descriptive file names. like guide-description-modal.tsx for quick searchability

Reply:
Will apply, thank you